akvo/rsr/views/project.py

```diff
@@ -413,6 +413,10 @@ def set_update(request, project_id, edit_mode=False, form_class=ProjectUpdateFor
     updates = project.updates_desc()[:5]
     update = None
 
+    # Prevent editing if project is completed or unpublished
+    if project.status == 'C' or not project.is_published():
+        raise PermissionDenied
+
     if update_id is not None:
         edit_mode = True
```

Comment:
It might be nice to disallow adding new updates, but allow to edit (or delete) existing updates. Just in case someone would like to change a spelling mistake or so, they should be able to.